zhaquirks/thirdreality/night_light.py

```diff
@@ -0,0 +1,106 @@
+"""Third Reality night light zigbee devices."""
+from zigpy.profiles import zha
+from zigpy.quirks import CustomCluster, CustomDevice
+from zigpy.zcl.clusters.general import (
+    Basic,
+    Groups,
+    Identify,
+    LevelControl,
+    MultistateInput,
+    OnOff,
+    Ota,
+    Scenes,
+    Time,
+)
+from zigpy.zcl.clusters.security import IasZone
+from zigpy.zcl.clusters.lighting import Color
+from zigpy.zcl.clusters.lightlink import LightLink
+from zigpy.zcl.clusters.manufacturer_specific import ManufacturerSpecificCluster
+from zigpy.zcl.clusters.measurement import (
+    OccupancySensing,
+    IlluminanceMeasurement,
+)
+
+from zhaquirks.const import (
+    DEVICE_TYPE,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    MODELS_INFO,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+    ZONE_STATUS,
+)
+from zhaquirks.thirdreality import THIRD_REALITY
+from zhaquirks import LocalDataCluster
+
+ThirdReality_Specific_CLUSTER_ID = 0xFC00
+
+class ThirdRealitySpecificCluster(CustomCluster):
+    """privatecluster, only used to relay ias zone information to ias zone Cluster."""
+
+    cluster_id = ThirdReality_Specific_CLUSTER_ID
+
+    def _update_attribute(self, attrid, value):
+        super()._update_attribute(attrid, value)
+        self.endpoint.ias_zone.update_attribute(IasZone.AttributeDefs.zone_status.id, value)
+
+
+class LocalIasZone(LocalDataCluster, IasZone):
+    """Local IAS Zone cluster."""
+
+    _CONSTANT_ATTRIBUTES = {
+        IasZone.attributes_by_name["zone_type"].id: IasZone.ZoneType.Motion_Sensor
```

Comment:
You can now also use `AttributeDefs` here (that's the newer way, instead of using `attributes_by_name`).
```suggestion
        IasZone.AttributeDefs.zone_type.id: IasZone.ZoneType.Motion_Sensor
```